process/sync/baseForkDetector.go

```diff
@@ -645,7 +646,7 @@ func (bfd *baseForkDetector) isConsensusStuck() bool {
 
 	lastCheckpointRound := bfd.lastCheckpoint().round
 	roundsDifference := bfd.roundHandler.Index() - int64(lastCheckpointRound)
-	if roundsDifference <= bfd.getMaxRoundsWithoutCommittedBlock(lastCheckpointRound) {
+	if roundsDifference <= bfd.getMaxRoundsWithoutCommittedBlock(uint64(bfd.roundHandler.Index())) {
```

Comment:
initially it was compared to `lastCheckpointRound`

Reply:
this was added in previous PR in `feat/sub-second-round`, i think it was added wrongly previously so i fixed it here, we should consider it based on current index